src/main/webapp/dev/js/common/visibilityOptions.es6

```diff
@@ -373,6 +373,9 @@ function updateVisibilityMessageDiv($containingForm) {
         type: 'POST',
         url,
         data: formData,
+        beforeSend() {
+            $visibilityMessageDiv.html("<img height='25' width='25' src='/images/ajax-preload.gif'/>");
```

Comment:
I prefer the `ajax-loader.gif` animation instead; it fits better into that part of the UI.